### PR TITLE
feature/asset-renaming

### PR DESCRIPTION
**Pull request by OmerGronich**
finished rename in s3
fixed "/" folder bug

### CODE REVIEW

**Review by davidmeirlevy — APPROVED** [review]

File: server/services/gcs.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const path = require('path');
-const uniqid = require('uniqid');
 const { joinUrl } = require('./url');
 const Gcs = require('../models/gcs');
+const { generateIdentifier } = require("./identifier");
```
Comment (drorrose) [inline server/services/gcs.js:4]: change double quotes to single quotes, just to use the same format as the rest of the code

File: server/services/s3.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 const S3 = require("../models/s3");
 const uniqid = require('uniqid');
 const path = require('path');
+const { generateIdentifier } = require("./identifier");
```
Comment (drorrose) [inline server/services/s3.js:4]: same comment about quotes, also should be changed in lines 1 and 5

**Review by drorrose — CHANGES_REQUESTED** [review]
Such minor changes..The rest of the code looks great!

File: server/services/gcs.js (hunk 1 above)
Comment (OmerGronich) [reply server/services/gcs.js]: done

File: server/services/s3.js (hunk 2 above)
Comment (OmerGronich) [reply server/services/s3.js]: done

**Review by drorrose — APPROVED** [review]

**Pull request merged**